build.gradle

```diff
@@ -184,7 +184,7 @@ dependencies {
   compile group: 'uk.gov.hmcts.reform', name: 'java-logging', version: versions.reformLogging
   compile group: 'uk.gov.hmcts.reform', name: 'java-logging-appinsights', version: versions.reformLogging
   compile group: 'uk.gov.hmcts.reform', name: 'service-auth-provider-client', version: '2.0.0'
-  compile group: 'uk.gov.hmcts.reform', name: 'idam-client', version: '0.0.2'
+  compile group: 'uk.gov.hmcts.reform', name: 'idam-client', version: '0.0.4'
   compile group: 'uk.gov.hmcts.reform', name: 'core-case-data-store-client', version: '4.5.0'
```

Comment:
Would be good to update `core-case-data-store-client` on a same go as well - it has lower version of feigns in there - jump from 9 -> 10 could potentially cause problems. Same goes for `service-auth-provider-client`